LICENCE-THIRD-PARTY

```diff
@@ -57,4 +679,104 @@ https://github.com/tweenjs/tween.js/
+uniseg
+------------------------------------------------------------------------------
+https://bitbucket.org/emptypage/uniseg-python
+Copyright 2013 Masaaki Shibata
+licensed under MIT licence.
+third-party-licences/uniseg.txt
+==============================================================================
+
+==============================================================================
+urijs
+------------------------------------------------------------------------------
+https://github.com/medialize/URI.js
+Copyright 2011 Rodney Rehm
+licensed under MIT licence.
+third-party-licences/urijs.txt
+==============================================================================
+
+==============================================================================
+Verto
+------------------------------------------------------------------------------
+https://github.com/uccser/verto
+Copyright University of Canterbury Computer Science Education Research and contributors.
+licensed under MIT licence.
+third-party-licences/verto.txt
+==============================================================================
+
+==============================================================================
+vinyl-buffer
+------------------------------------------------------------------------------
+https://github.com/hughsk/vinyl-buffer
+Copyright 2014 Hugh Kennedy
+licensed under MIT licence.
+third-party-licences/vinyl-buffer.txt
+==============================================================================
+
+==============================================================================
+viz-lite.js
+------------------------------------------------------------------------------
+Source file obtained from
+https://github.com/mdaines/viz.js
+Copyright 2014-2017 Michael Daines
+licensed under MIT licence.
+third-party-licences/vizlite.txt
+==============================================================================
+
+==============================================================================
+wheel
+------------------------------------------------------------------------------
+https://github.com/pypa/wheel
+Copyright 2012-2014 Daniel Holth and contributors
+licensed under MIT licence.
+third-party-licences/wheel.txt
+==============================================================================
+
+==============================================================================
+Whoosh
+------------------------------------------------------------------------------
+https://bitbucket.org/mchaput/whoosh/overview
+Copyright 2011 Matt Chaput
+licensed under BSD licence.
+third-party-licences/wheel.txt
+==============================================================================
+
+==============================================================================
+wnumb
+------------------------------------------------------------------------------
+https://github.com/leongersen/wnumb
+Created by Leon Gersen
+licensed under WTFPL
```

Comment:
Should we add 'No known licence file to include' here like you have elsewhere?

Reply:
WTFPL doesn't really have a licence per se
http://www.wtfpl.net/
Should I do it anyway?